### DISCUSSION TITLE
[Bug] Reverse arrows appearing at wrong place in editor

### DISCUSSION THREAD

**Discussion by hwabis** [ppy/osu#26759] [Q&A] [answered]
https://github.com/ppy/osu/assets/62194990/012dd442-7ea9-41ef-8445-1fb48e0d5381

[1706450408.runtime.log](https://github.com/ppy/osu/files/14076309/1706450408.runtime.log)

Here is also an export of the beatmap: 
[Sparxe - Dream Elevator.zip](https://github.com/ppy/osu/files/14076318/Sparxe.-.Dream.Elevator.zip)

Using argon skin in video but also happens for argon pro, triangles, and classic

**Comment by peppy** [comment, marked as answer]
already fixed in next release.
